test/topology/conftest.py

```diff
@@ -5,7 +5,7 @@
 #
 # This file configures pytest for all tests in this directory, and also
 # defines common test fixtures for all of them to use
-
+import asyncio
```

Comment:
Is this change connected with this PR?

Reply:
The same situation here as with importing re. Getting reed of the errors complained by pycharm.